serverlessworkflow/modules/ROOT/pages/tooling/quarkus-dev-ui-extension/quarkus-dev-ui-overview.adoc

```diff
@@ -0,0 +1,45 @@
+= Kogito Serverless Workflow Tools - Quarkus Dev UI
+
+Quarkus applications comes with an interactive Dev UI, that provides developers with a number of tools to help in the process of development. You can plug in these tools to a Quarkus application in the form of extensions. The Kogito Serverless Workflow Tools is an extension, that provides a Console for you to view, manage and start Workflows Instances. You can add this extension as a _dependency_ to the _pom.xml_ of your Kogito Project.
+
+.Kogito Serverless Workflow Tools dependency in pom.xml
+[source,xml]
+----
+<dependency>
+  <groupId>org.kie.kogito</groupId>
+  <artifactId>kogito-quarkus-serverless-workflow-devui</artifactId>
+  <version>${VERSION}</version>
+</dependency>
+----
+
+Another dependency for the Console is to add the _kogito-addons-quarkus-process-svg_ to provide the SVG diagram to the Consoles.
+
+.Quarkus Addons:Process-SVG dependency in pom.xml
+[source,xml]
+----
+<dependency>
+  <groupId>org.kie.kogito</groupId>
+  <artifactId>kogito-addons-quarkus-process-svg</artifactId>
+</dependency>
+----
+
+.The Quarkus application can be started by running the following command:
+[source,shell]
+----
+mvn clean quarkus:dev
+----
+
+The Quarkus Dev UI can be accessed by using `http://localhost:8080/q/dev/`
+
+.The Kogito Serverless Workflow Tools Extension in Quarkus Dev UI
+image::tooling/quarkus-dev-ui-extension/kogito-swf-tools-quarkus-dev-ui.png[]
+
+The Kogito Serverless Workflow Tools tab of the Dev UI shows the number of *Workflow Instances* available. Clicking on it will navitage you to the Console page.
```

Comment:
```suggestion
The Kogito Serverless Workflow Tools tab of the Dev UI shows the number of *Workflow Instances* available. Clicking on it will navigate you to the Console page.
```